spinalcordtoolbox/scripts/sct_compute_snr.py

```diff
@@ -120,6 +127,12 @@ def main(argv=None):
         index_vol_user = arguments.vol
     else:
         index_vol_user = ''
+    
+    #Added function for text file 
+    if arguments.o is not None:
+        file_name = arguments.o
+
+
```

Comment:
```suggestion
    if arguments.o is not None:
        file_name = arguments.o
    else:
        file_name = None

```

Reply:
If the default value for `arguments.o` is `None`, I think we can just do `file_name = arguments.o` and it will have the same effect.

Reply:
yes, i was not sure it would work (the problem with remote forks is that we cannot commit the changes and immediately test them)-- but yes, let's do it that way-- thank you

Reply:
> (the problem with remote forks is that we cannot commit the changes and immediately test them)

That's fair! Yeah, there are only a couple of ways to do this:

* Create a local branch that points to the fork's branch, then make changes locally and test them locally.
* Or, if @santoshdvc has checked the box [**"Allow edits from maintainers"**](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests)), then we can make changes directly to the fork's branch.

Reply:
> Or, if @santoshdvc has checked the box "Allow edits from maintainers"), then we can make changes directly to the fork's branch.

oh! great feature! @santoshdvc could you please do this?